.golangci.yml

```diff
@@ -0,0 +1,9 @@
+run:
+  build-tags:
+  - e2e
+  skip-dirs:
+  - vendor
+  - pkg/client/clientset/versioned/fake
+linters-settings:
+  errcheck:
+    exclude: .errcheck.txt
```

Comment:
ooo what's this file for? ive never seen this before! :D

(question about both the .golangci.yaml and the .errcheck.txt!)

Reply:
This is the config file for our linter `golangci-lint` (more info [here](https://github.com/golangci/golangci-lint#config-file)). `.errcheck.txt` lists files to exclude from testing. However, I don't think we need `.errcheck.txt` anymore, because the linter error we were skipping was fixed [the correct way](https://github.com/tektoncd/triggers/pull/21#discussion_r305095390) in @vincent-pli's webhook PR 😅 

I'm going to remove the `.errcheck.txt` file in a PR that I'm putting together, because I don't think we need it anymore.